src/coreclr/jit/codegencommon.cpp

```diff
@@ -5078,6 +5492,9 @@ void CodeGen::genZeroInitFltRegs(const regMaskTP& initFltRegs, const regMaskTP&
 #elif defined(TARGET_ARM64)
                 // We will just zero out the entire vector register. This sets it to a double/float zero value
                 GetEmitter()->emitIns_R_I(INS_movi, EA_16BYTE, reg, 0x00, INS_OPTS_16B);
+#elif defined(TARGET_LOONGARCH64)
+                // We will just zero out the entire vector register. This sets it to a double/float zero value
```

Comment:
It's not called a "vector register" on LA, is it?

Reply:
Yes, LA does not suppport SIMD right now.
We will submit SIMD feature in future seperately.